support-frontend/app/utils/CheckoutValidationRules.scala

```diff
@@ -180,6 +180,17 @@ object AddressAndCurrencyValidationRules {
     ) {
       stateFromRequest.isDefined.otherwise(s"state is required for $countryFromRequest")
     } else Valid
+  def hasValidPostcodeLength(postcodeFromRequest: Option[String], addressType: String): Result = {
```

Comment:
What would you think about naming this more specifically? Something like `postcodeIsShortEnoughForSalesforce`. I know it’s a bit long, but I think it’d be useful for it to be clear from the code (both definition and uses) that this is why we’re limiting it to 20 characters.

Reply:
I have kept hasValidPostcodeLength in order to match with the other functions  in the same file.I have  added a comment to make it clear

Reply:
Ok, that sounds good :)